src/xwayland-unmanaged.c

```diff
@@ -119,7 +119,11 @@ unmanaged_handle_override_redirect(struct wl_listener *listener, void *data)
 static void
 unmanaged_handle_request_activate(struct wl_listener *listener, void *data)
 {
-	wlr_log(WLR_DEBUG, "request_activate not handled\n");
+	struct xwayland_unmanaged *unmanaged =
+		wl_container_of(listener, unmanaged, request_activate);
+	wlr_xwayland_surface_activate(unmanaged->xwayland_surface, true);
```

Comment:
Do we have to check for `unmanaged->xwayland_surface` here? Not sure what happens on unmap.

Reply:
Addressed in v2 by doing the restack in `unmanaged_handle_map()`.